src/adapters/prebidServer.js

```diff
@@ -58,12 +58,25 @@ function PrebidServer() {
             if(bidder.no_bid) {
               // store a "No Bid" bid response
 
-              let bidObject = bidfactory.createBid(STATUS.NO_BID, {
-                bidId: bidder.bid_id
-              });
-              bidObject.adUnitCode = bidder.ad_unit;
-              bidObject.bidderCode = bidder.bidder;
-              bidmanager.addBidResponse(bidObject.adUnitCode, bidObject);
+              if (!bidder.ad_unit) {
+                utils.getBidderRequestAllAdUnits(bidder.bidder).bids.forEach(bid => {
+                  let bidObject = bidfactory.createBid(STATUS.NO_BID);
```

Comment:
Pass through the `bid` object as the 2nd param to `createBid` so we can get the ID (see line 78 below as an example)